makefile.update.kw.inc

```diff
@@ -5,19 +5,23 @@
  * This file contains the functions that are required to execute
  * `drush kw-update-makefile`.
  */
-
+use Drush\UpdateService\ReleaseInfo;
 /**
  * Implements drush_COMMAND() for `drush kw-update-makefile`.
- * 
+ *
  * @param string $location
  *   The location of the make file to be updated.
  */
 function drush_kraftwagen_kw_update_makefile($makefile) {
   require_once dirname(__FILE__) . '/includes/kraftwagen.info_files.inc';
 
   $info = kraftwagen_read_info_file($makefile);
-  $update_info = kraftwagen_update_projects($info);
-
+  if (version_compare(drush_core_version(), '7', '>=')) {
+    $update_info = kraftwagen_update_projects78($info);
+  }
+  else {
+    $update_info = kraftwagen_update_projects($info);
```

Comment:
Alternatives: put this before the `if` so that it is always set and only changed when the condition is met, or use a ternary.